test/unit/services/instanceAdapter_spec.js

```diff
@@ -217,32 +217,4 @@ describe("The InstanceAdapter class ", function () {
 			});
 		});
 	});
-
-	describe("updating an already created instance", function () {
```

Comment:
Can you explain why this test case is being removed? It seems unrelated to the SSH changes and removing test cases is usually not a great idea.